A58-client-side-weighted-round-robin-lb-policy.md

```diff
@@ -2,9 +2,9 @@ A58: `weighted_round_robin` LB policy
 ----
 * Author(s): @yousukseung
 * Approver: @markdroth
-* Status: Draft
-* Implemented in: -
-* Last updated: 2023-02-05
+* Status: Implemented
+* Implemented in: C-core, Java
```

Comment:
Isn't go also implemented? https://github.com/grpc/grpc-go/tree/master/balancer/weightedroundrobin looks like impelmentation of this.

Reply:
@dfawley @zasweq we we ready to add Go here?

Reply:
Yeah it's done. There's a few pending bug fixes and linkage into xDS prs nut other than that it's done.

Reply:
Added Go.